ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -117,3 +117,31 @@ static_library('nnstreamer_decoder_image_segment',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
+# protobuf decoder
+if protobuf_dec_support_is_available
+  pb_gen = generator(pb_comp, output: ['@BASENAME@.pb.h', '@BASENAME@.pb.cc'], arguments : ['--proto_path=@CURRENT_SOURCE_DIR@/../include/', '--cpp_out=@BUILD_DIR@', '@INPUT@'])
+  pb_gen_src = pb_gen.process('../include/nnstreamer.proto')
```

Comment:
How about using meson.current_source_dir()?

Reply:
`protoc` complains (and do not run) when absolute path and relative path both are given. I think the current `'./include/nnstreamer.proto'` would be more concise.